.github/workflows/build.yml

```diff
@@ -117,7 +117,7 @@ jobs:
 
       - name: Cargo E2E Tests
         run: |
-          make test-e2e
+          make test-fermyon-platform
```

Comment:
Can we update the `name:` string above as well?  Perhaps `Fermyon Platform Integration Tests` or similar?

Reply:
thanks Vaughn. updated